host/xtest/pkcs11_1000.c

```diff
@@ -22,9 +22,18 @@
 
 #include "xtest_test.h"
 #include "xtest_helpers.h"
+#include "xtest_uuid_helpers.h"
 
 #include <regression_4000_data.h>
 
+/*
+ * AUTH_TYPE enumerates the types of user authentication
+ *
+ * PIN_AUTH  Pin authentication.
+ * ACL_AUTH  ACL authentication.
+ */
+typedef enum { PIN_AUTH, ACL_AUTH } AUTH_TYPE;
```

Comment:
Actually I'd rather have `enum auth_type { PIN_AUTH, ACL_AUTH };` because we prefer to avoid typedefs, and types should be lower case (unless from some standard API). Sorry for my previous incomplete comment on this.